app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -0,0 +1,46 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+const createSchema = (fields, promise, edit, loadSchema) => ({
```

Comment:
This function needs to be converted to the block-return form in order to declare an additional helper constant:
```
const createSchema = (fields, promise, edit, loadSchema) => {
  const idx = fields.findIndex((field) => field.name === 'volume_type');

  return ({
    fields: [/* ... */],
  });
};
```

This `idx` will be necessary for the `resolveProps` below...